### DISCUSSION TITLE
How to remove Metadata from Lora files?

### DISCUSSION THREAD

**Discussion by le-khang** [kohya-ss/sd-scripts#784] [General]
Hi Kohya,
Thank you for making this awesome training tool. May I ask is there any way to remove metadata from LORA files? I kinda don't want my friends or clients know how I've described them in the dataset.
![2023-08-22 14_13_43-Stable Diffusion](https://github.com/kohya-ss/sd-scripts/assets/35536989/b59b99fc-8a58-42b7-ba3d-e16b2c339c78)

**Comment by le-khang** [comment]
Or could you please make it so including metadata is optional?

**Comment by CCpt5** [comment]
Prob not really the answer you're looking for, but I was looking at this repo last night ("Ai-Tools" for LoRA) and the LoRA rescale .yaml you edit the values on has a section for metadata which you can change/etc. Requires a little bit of python command line knowledge:

https://github.com/ostris/ai-toolkit

**Reply by le-khang** [reply]
Thank you very much but I'm not a coder so I'm not sure how to use that. I guess I have to find other solution like merging/extracting Lora or something.
